homeassistant/components/mqtt/alarm_control_panel.py

```diff
@@ -46,7 +44,7 @@
 from .debug_info import log_messages
 from .mixins import (
     MQTT_AVAILABILITY_SCHEMA,
```

Comment:
Should this also be moved to MQTT_ENTITY_BASE_SCHEMA?

Reply:
This would also make it possible to override these values. If that's OK i can't see any problem. Let me know.

Reply:
You're right, and that's already an issue for the `CONF_DEVICE` and `CONF_UNIQUE_ID` which we don't want the components to override.

Can we do like this to prevent the overriding, and remove `CONF_NAME` from the `MQTT_ENTITY_BASE_SCHEMA`?

```py
PLATFORM_SCHEMA = (
    mqtt.MQTT_BASE_PLATFORM_SCHEMA.extend(
        {
            # component schema
        }
    )
    .extend(MQTT_ENTITY_BASE_SCHEMA.schema)
)
```

Reply:
Sure, we can, provided components never need to override `CONF_ICON` (I guess domain default icons are fine).

And dropping `CONF_ICON` would make it default to something more sensible, such as "mqtt alarm" as mentioned in https://github.com/home-assistant/core/pull/47165#discussion_r587263324

I'll get to it and let you know! :)